src/main/java/org/generationcp/middleware/dao/dms/DmsProjectDao.java

```diff
@@ -1457,11 +1458,11 @@ public long countStudies(final StudySearchFilter studySearchFilter) {
 		return ((BigInteger) sqlQuery.uniqueResult()).longValue();
 	}
 
-	public List<StudyDto> getStudies(final StudySearchFilter studySearchFilter) {
+	public List<StudyDto> getStudies(final StudySearchFilter studySearchFilter, final PageRequest pageRequest) {
```

Comment:
Better use the `final Pageable pageable` in this DAO class to be consistent with other classes